### PR TITLE
change Bug template & PR template

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`npm run test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

#### Description of changes

Change Bug and PR template as discussed to be more clear and helpful.

List of changes:
- Ask for CodePen example in Bug template
- Reorder PR template to have description first, then Notes, then PR Checklist

#### Bug template preview screenshot:

image

#### PR template preview screenshot:

image

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-- [ ] Added relevant unit test for your changes. (`npm run test`)
-- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
-- [ ] Ran precheckin (`npm run precheckin`)
-- [ ] Added screenshots/GIFs for UI changes.
-
 #### Description of changes
 
 (give an overview. Add technical description describing your changes)
 
 #### Notes for reviewers
```
Comment (dbjorge) [inline .github/pull_request_template.md]: I thought we agreed the "Notes for reviewers" section as well; did I misremember?

Comment (Shobhit1) [reply .github/pull_request_template.md]: Updated!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -19,6 +19,13 @@ Steps to reproduce the behavior:
 3. Scroll down to '...'
 4. See error
 
+## Code Pen Repro Example
```
Comment (waabid) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This should be in sentence case.

Comment (waabid) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Code Pen => CodePen

Comment (waabid) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## CodePen repro example
```

**Pull request merged**